cypress/e2e/po/components/shell.po.ts

```diff
@@ -0,0 +1,19 @@
+import ComponentPo from '@/cypress/e2e/po/components/component.po';
+
+export default class Shell extends ComponentPo {
+  constructor() {
+    super('#windowmanager');
+  }
+
+  readonly terminalRow: string = '.xterm-rows'
+
+  openTerminal() {
+    // get and click on the first row's action menu button
+    cy.get(`button[data-testid="sortable-table-0-action-button"]`).click();
```

Comment:
@Sean-McQ @rak-phillip We shouldn't be using `cy.get` directly for these elements. There's a SortableTablePo which has handlers to execute actions for rows.